registry/remote/repository.go

```diff
@@ -506,6 +506,7 @@ func (s *blobStore) Push(ctx context.Context, expected ocispec.Descriptor, conte
 	if err != nil {
 		return err
 	}
+	hs := strings.Split(req.URL.Host, ":")
```

Comment:
The name `hs` is confusing. We need a more descriptive name as this variable is used in 20 lines later. How about `reqHost`?